ignite/config/plugins/config.go

```diff
@@ -35,7 +35,9 @@ type Plugin struct {
 	// path: github.com/foo/bar/plugin1@v42
 	Path string `yaml:"path"`
 	// With holds arguments passed to the plugin interface
-	With map[string]string `yaml:"with,omitempty"`
+	With map[string]string `yaml:"with"`
+	// SharedHost enables sharing a single plugin server across all running instances
```

Comment:
All running instances of what ? 

I'm sorry but I think I still don't totally understand this feature :P

Reply:
If the `Sharehost` flag is enabled other plugins with the same `Path` from the same `plugins.yml` only a single instance of a rpc server will be created for that plugin. This allows plugin server's that spawned from long running processes like `ignite chain serve` to be shared by other commands that are run from **the same** directory (the same config context).
This is nice if you want to interact with state held in a single instance from multiple command contexts. this can help developers write more plugin commands that interact on a shared state of a plugin host.

Reply:
Got it now thanks, this will be useful for sure.